packages/wonder-blocks-cell/src/components/internal/cell-core.tsx

```diff
@@ -172,6 +172,7 @@ const CellCore = (props: CellCoreProps): React.ReactElement => {
         href,
         onClick,
         "aria-label": ariaLabel,
+        "aria-selected": ariaSelected,
```

Comment:
**note:** This is needed by `OptionItem` so we can know when a given option is selected.